index.js

```diff
@@ -14,6 +14,24 @@ cmdShim.ifExists = cmdShimIfExists
 
 /**
  * @typedef {import('./index').Options} Options
+ *
+ * @typedef {object} RuntimeInfo Infomation of runtime and its arguments
```

Comment:
Information*

Reply:
That was there in #12.

That said. I’ll fix it and other typos.